src/bots/groq/Gemma29bGroqAPIBot.js

```
@@ -0,0 +1,10 @@
+import GroqAPIBot from "./GroqAPIBot";
+
+export default class Gemma29bGroqAPIBot extends GroqAPIBot {
+  static _className = "Gemma29bGroqAPIBot";
+  static _logoFilename = "gemma2-9b-it-groq-logo.png";
+  static _model = "gemma2-9b-it";
+  constructor() {
+    super();
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)